app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,7 @@
   </head>
 
   <body>
+  <%= render 'layouts/header', user: @current_user %>
   <section class="main_section">
     <div class="container">
     <%= yield %>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169177198-b9dfac48-d08b-436b-b604-28903ec4a875.png)

## [OPTIoNAL]
- [ ]  Kindly would you please follow the [wireframes.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md) and add some styling to your project. I know that the styling does not matter for now but it will give your project a better structure and UI. I know you can do it :muscle: :muscle: